internal/jobs/transform.go

```diff
@@ -142,14 +142,20 @@ func (s *Scheduler) parseTransform(config *JobConfiguration) (Transform, error)
 		if transformTypeName != nil {
 			if transformTypeName == "HttpTransform" {
 				transform := &HttpTransform{}
-				endpoint, ok := transformConfig["Url"]
-				if ok && endpoint != "" {
-					transform.Endpoint = endpoint.(string)
+				url, ok := transformConfig["Url"]
+				if ok && url != "" {
+					transform.Url = url.(string)
 				}
 				tokenProvider, ok := transformConfig["TokenProvider"]
 				if ok {
 					transform.TokenProvider = tokenProvider.(string)
 				}
+				timeout, ok := transformConfig["TimeOut"]
+				if ok && timeout != 0 {
+					transform.TimeOut = timeout.(float64)
+				} else {
+					transform.TimeOut = 0
```

Comment:
I suppose timeout=0 means indefinite? Should we make timeout mandatory for httpTransform config instead, so that users have to explicitly set it to 0?